src/authorizations/components/TokenList.tsx

```diff
@@ -45,6 +51,7 @@ export default class TokenList extends PureComponent<Props, State> {
   public currentPage: number = 1
   public rowsPerPage: number = 10
   public totalPages: number
+  private enableBulkActionDelete = isFlagEnabled('bulkActionDelete')
```

Comment:
What’s the point of having this be a property of the class? Why not just use the isFlagEnabled instead?